pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -25,6 +25,8 @@ buildDotnetModule (finalAttrs: {
     hash = "sha256-FzQphMhiC1g+6qmk/R1v4rq2ldy35NcaWm0RR1UlwLA=";
   };
 
+  enableParallelBuilding = false;
```

Comment:
I assume you tried to build in parallel but it failed?

Reply:
It was in the first nixpkgs-review, on aarch64-linux.  Something we should look into.